lib/components/chat_history/auxiliary/realtimecash_donation.dart

```diff
@@ -17,11 +17,13 @@ class RealtimeCashDonationEventWidget extends StatelessWidget {
         Text.rich(
           TextSpan(
             children: [
-              const TextSpan(text: "You received "),
+              TextSpan(text: "${model.donor} donated "),
               TextSpan(
-                  text: "${model.value.toString()} ${model.currency}.",
+                  text: "${model.value.toString()} ${model.currency}. ",
                   style: boldStyle),
-              const TextSpan(text: " Your transaction hash: "),
+              if (model.message.isNotEmpty)
+                TextSpan(text: "message: ${model.message}"),
+              const TextSpan(text: " Hash: "),
```

Comment:
We can probably omit the hash. It's not really useful information. Maybe a useful way to use it would be long press opens up etherscan/polygonscan?

Reply:
This can be done in a separate PR, but I think we should remove the hash.